packages/ui/src/accounts/components/Accounts.tsx

```diff
@@ -29,10 +31,6 @@ export function Accounts() {
     [visibleAccounts, balances, sortBy, isDescending]
   )
 
-  if (!hasAccounts) {
-    return <Loading />
-  }
```

Comment:
IMO it would be an easier pattern to follow for the community if: 
```jsx
<ListItemLoader columnsTemplate="276px repeat(4, 128px) 104px" count={4}>
  ...
</ListItemLoader>
```
Was moved in a separate component called e.g `AccountListLoading` and called here, although you're right `!hasAccounts` should be `isLoading`.

This way the community developer will just replace the `<Loading>` components by `<[Custom]Loading>` components created, and it will be easier for us to review too